sdk/identity/identity/src/credentials/interactiveBrowserCredential.ts

```diff
@@ -186,7 +186,7 @@ export class InteractiveBrowserCredential implements TokenCredential {
 
         try {
           const authResponse = await this.pca.acquireTokenByCode(tokenRequest);
-          res.sendStatus(200);
+          res.status(200).send("Authentication Complete. You can close the browser and return to the application.");
```

Comment:
Can we make a PR for just this piece? I've updated some of the success messages in https://github.com/Azure/azure-sdk-for-js/pull/11871, can you check that against this PR to see what still needs to be updated?

Reply:
#11871 looks good. The caveat is that the formatSuccess method that we have uses `SUCCESS: ____` instead of `Authentication Complete`. Which one should we use, @g2vinay ?

Reply:
I'd say that the message should be consistent everywhere. Second the logging message should offer some more detail related to the status, for e.g. Successfully retrieved the access token for {Scopes}, Authentication Failed / Failed to retrieve access token for {Scopes}. These are just examples.

Does the logger in JS add more info underneath automatically beyond the message SUCCESS: {Scopes} or ERROR: {Scopes}

@schaabs

Reply:
No, the logger doesn't do any magic underneath!